docs/guides/kubernetes/securing-an-LKE-cluster/index.md

```diff
@@ -0,0 +1,292 @@
+
+1. Create a role.yaml file with the following contents:
+
+{{< file >}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+ name: example-role
+ namespace: examplenamespace
+rules:
+ - apiGroups: [""]
+   resources: ["pods"]
+   verbs: ["get", "watch", "list"]
+{{< /file >}}
+
+The above example would allow any user with the assigned role to `get`, `watch`, and `list` resources in the `examplenamespace` namespace. The `name` `example-role` is a unique identifier which can be called when applying the `rolebinding` in the next step.
+
+1. Once the role is created, create a `rolebinding.yaml` file to bind the role to your user:
+
+{{< file >}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+ name: example-role-binding
+ namespace: default
+subjects:
+ - apiGroup: rbac.authorization.k8s.io
+   kind: User
+   name: exampleuser
+roleRef:
+ apiGroup: rbac.authorization.k8s.io
+ kind: Role
+ name: example-role
+{{< /file >}}
+
+1. Apply both the `role.yaml` and the `rolebinding.yaml` to grant permission to the new user:
+
+       kubectl apply -f role.yaml
+       kubectl apply -f rolebinding.yaml
+
+1. To test, switch the active kubeconfig and namespace to the ones created for the limited example user:
+
+       export KUBECONFIG=exampleuser_kubeconfig.yaml
+       kubectl config set-context --current --namespace=examplenamespace
+
+1. If the configuration works, you should not see any errors when requesting information on active pods in the namespace:
+
+       kubectl get pods
+
+  However, if the user attempts to get information on nodes, or any other request that has not been explicitly configured, the request will fail with an error similar to the following:
+
+  {{< output >}}
+  Error from server (Forbidden): nodes is forbidden: User "exampleuser" cannot list resource "nodes" in API group "" at the cluster scope
+  {{< /output >}}
+
+
+## Next Steps
+
+Now that the user has been successfully installed, the users `kubeconfig` file may be exported for other users to use from their own `kubectl` clients, and the user can access the cluster with the limited permissions set by the administrator in their own namespace. Additionally security controls may still be applied, however will vary depending on your use case. **Admission Controllers** for example, are a great way to implement additional controls on authenticated and authorized requests. Apllications on an LKE cluster can additionally be put behind a NodeBalancer and ingress with TLS enabled.  For more information, the following resources may be helpful:
```

Comment:
"Now that the user has been successfully installed, the **user's**..." (added apostrophe)